### PR TITLE
Solar assembly crate buff

### PR DESCRIPTION
**Pull request by K-Dynamic**
## About the PR

- Solar assembly crate contents changed from 6 solar assembly flatpacks to 10 flatpacks, 20 sheets of glass
- Crate cost increased from 525 to 1250 spesos to compensate

## Why / Balance

Engies can't expand or repair solars with only 6 flatpacks per crate, as well as lacking glass on hand. It would be like ordering a door kit but only having door electronics.

Engies will probably forget about this buff so I will request mappers to add solar crates to some solar airlocks and an extra in engineering. (As well as potentially downscaling some solars so engis learn about solar assembly.)

## Technical details

yml changes only
- `Resources/Prototypes/Catalog/Cargo/cargo_engines.yml` (price increase from 525 to 1250 spesos)
- 'Resources/Prototypes/Catalog/Fills/Crates/engines.yml' (solar crate contents and description changed)

## Media

image
image

## Requirements

- [x] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [x] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes

N/A

**Changelog**

:cl:
- tweak: Solar assembly crate now comes with 10 flatpacks and 20 glass to make expansion and repairs easier, as well as increasing in price from 525 to 1250 spesos.

### CODE REVIEW

**Comment by thebadman4662** [comment]
Glass crates are still 90 sheets though? Its plasma crate that got nerfed down to 33% capacity.

**Comment by pheenty** [comment]
> Glass crates are still 90 sheets though? Its plasma crate that got nerfed down to 33% capacity.

Yes they are

And 525 / 6 *  10 + 1500 / 90 * 20 = 1208

So 1200 was basically perfect, 1700 is too much

**Comment by pheenty** [comment]
I mean that's fine but still i liked 1200 more

File: Resources/Prototypes/Catalog/Fills/Crates/engines.yml [hunk 1]
```diff
@@ -120,12 +120,14 @@
   id: CrateEngineeringSolar
   parent: CrateEngineering
   name: solar assembly crate
-  description: Parts for constructing solar panels and trackers.
+  description: A kit with solar flatpacks and glass to construct ten solar panels and trackers.
```
Comment (slarticodefast) [inline Resources/Prototypes/Catalog/Fills/Crates/engines.yml:123]:
```suggestion
  description: A kit with solar flatpacks and glass to construct ten solar panels.
```
Tracker is not included, it needs special electronics to build (even though it currently does literally nothing).

**Review by slarticodefast — APPROVED** [review]
I was against including the glass sheets at first since those can already be ordered separately, but K-Dynamic made some good points about potentially mapping the crate, so having the glass included is useful. The price looks good to me:
(5225 / 6 * 10)+1500 / 90 * 20=1208
Adjusts for the increase in flatpacks and adds the price of the glass.

**Comment by chromiumboy** [comment]
Seems fine to me

**Pull request merged**